backend/src/database/migrations/20220711025216-update-is-credential-used-function.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, _) => {
+    await queryInterface.createFunction(
```

Comment:
I presume creating a new function with the same name will override the previous function?

Reply:
yup

Reply:
> oh, might be a good idea to include a comment in the previous database migration to say that it's outdated.

haha think it's still good to this, just to avoid future confusion